sdk/src/constants.ts

```diff
@@ -8,7 +9,7 @@ import {toUtf8Bytes} from '@ethersproject/strings';
 
 /** Timeout that will be applied to operations involving
  * many fetch requests that could take a long time */
-export const MULTI_FETCH_TIMEOUT = 7 * 1000;
+export const MULTI_FETCH_TIMEOUT = 7 * TIME.SECOND_MS;
```

Comment:
Does it make sense to have a constants file with the new approach you are taking to order the code?

Reply:
I think this `constants.ts` file should be split into smaller files bundling specific functionalities.